core/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -63,7 +105,9 @@ case class Newlines(
     penalizeSingleSelectMultiArgList: Boolean = true,
     alwaysBeforeCurlyBraceLambdaParams: Boolean = false,
     alwaysBeforeTopLevelStatements: Boolean = false,
-    afterCurlyLambda: NewlineCurlyLambda = NewlineCurlyLambda.never
+    afterCurlyLambda: NewlineCurlyLambda = NewlineCurlyLambda.never,
+    afterImplicitKWInVerticalMultiline: Boolean = false,
+    beforeImplicitKWInVerticalMultiline: Boolean = false
```

Comment:
in #735 @pjan pointed out that this is something he would like to be supported, I also saw this style used in some code bases (e.g. ensime). I don't have a strong opinion on this myself.